sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 - Text analytics SDK update the service to version `v3.0` from `v3.0-preview.1`.
```

Comment:
could we add the version the library supports to the Readme file too? Kind of what we did in FR: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/formrecognizer/azure-ai-formrecognizer#include-the-package

Reply:
yes i can